src/main/resources/config.yml

```diff
@@ -5,3 +5,4 @@ cmdsvr_port: 4731
 pyexe: "/usr/bin/python3"
 # windows pyexe example
 # pyexe: "C:\\Users\\wensheng\\Anaconda3\\python.exe"
+console-command-whitelist:
```

Comment:
We could discuss here about a default value, even if its empty, but i think it is also ok like it is now

Reply:
That would be an empty list... seems I can use JSON syntax (`[]`) in YAML for that.